README.md

```diff
@@ -2,6 +2,18 @@
 
 A CLI application to generate and email credit card slips for Alma invoices via the Alma API.
 
+## Description
+Credit card slips are generated for items purchased with a credit card (e.g. from vendors like Amazon), which means we prepay rather than receive an invoice after shipment, which is our workflow with most vendors. The credit card slip is in lieu of a vendor-generated invoice, and is used for processing by Acquisition staff.
+
+ The application runs daily and retrieves PO lines with:
```

Comment:
Does "PO" stand for "Purchase Order"?  Not sure it needs spelling out, maybe obvious in context, but just curious.

Reply:
Correct! And yes, I think the context would be enough for future readers